lib/parser.js

```diff
@@ -75,8 +75,10 @@ Parser.prototype._preprocess = function (element, context, config) {
         self._fileCache[filePath] : fs.readFileSync(filePath, 'utf8');
     } catch (e) {
       // Read file fail
+      let missingReferenceError = ` Missing reference in: ${element.attribs[ATTRIB_CWF]}`;
```

Comment:
- Remove space in <code>` M</code >
- Rename to `missingReferenceErrorMessage`
- How about <code>e.message += \`\n${missingReferenceErrorMessage}`;</code>?